### DISCUSSION TITLE
Adding and Subtracting Series objects

### DISCUSSION THREAD

**Discussion by CommonClimate** [LinkedEarth/Pyleoclim_util#592] [Ideas]
I recently got a question about how to subtract two `Series` (h/t @shiness11). My response was the following:

> We don’t want to make it too easy to subtract Series objects, because in general they won’t have the same units or time axis, so it doesn’t make sense except in very special circumstances.
(then a bunch of somewhat convoluted explanations for how to do it followed). 

Upon reflection, it seems reasonable to me to check internally that two series have the same units and time axis, and (if not), to standardize them and put them on a common time axis prior to adding or subtracting their values.  We can debate what the default behavior should be, but I don't have the same reservations about the idea than the QuanSight folks did, and I think our users are entitled to that level of service! 

@khider what do you think?

**Comment by khider** [comment]
Can you give me a use case on why you would want to subtract two series?

**Comment by CommonClimate** [comment]
Lots of examples. Could be to compute spatial gradients, remove a forcing (e.g. scaled insolation) from a signal to highlight internal variability, or remove the thermal component from d18O to get d18O_sw, to name a few

**Comment by CommonClimate** [comment]
Also, synthetic records like [this one](https://www.science.org/doi/abs/10.1126/science.1203580) involve scaling, adding or subtracting series

**Comment by khider** [comment]
A lot of these applications wouldn't require the same units and it's more a workflow than a function. I would suggest we write a tutorials and show how to access the parameters directly for things like this.

**Comment by khider** [comment]
And I would suggest adding a use case like this to PaleoBooks

**Comment by CommonClimate** [comment]
Fair enough - I added this to [this list](https://github.com/LinkedEarth/PaleoBooks/issues/4)
